bin/client_package/installation/post-uninstall.sh

```diff
@@ -1,21 +1,21 @@
 #!/bin/bash
 
 function disable_systemd {
-    printf  "Disabling systemd script '/lib/systemd/system/mysterium-client.service'..\n" \
-        && systemctl disable mysterium-client \
-        && rm -f /lib/systemd/system/mysterium-client.service
+    printf  "Disabling systemd script '/lib/systemd/system/mysterium-client.service'..\n"
+    systemctl disable mysterium-client
```

Comment:
no, this is exact the problem, sometimes, when you try to remove package or purge, it may fail due to how dpkg works. It may try to execute removal scripts twice producing and error and leaving package in inconsistent state.